tests/test_integration.py

```diff
@@ -1170,8 +1174,13 @@ def test_setup_python_multiple_direct_markers():
 
     py2_only_program = [pex, '-c', 'import subprocess32']
 
-    with pytest.raises(subprocess.CalledProcessError):
-      subprocess.check_call(py2_only_program, env=make_env(PATH=os.path.dirname(py36_interpreter)))
+    with pytest.raises(subprocess.CalledProcessError) as err:
+      subprocess.check_output(py2_only_program, stderr=subprocess.STDOUT,
+                              env=make_env(PATH=os.path.dirname(py36_interpreter)))
+    # TODO: Is the intention of this test that this should fail on platform incompatibility?
```

Comment:
It is not, this exposes a bug: #899. Thank you!

There is also a very similar bug described in #898 found as a result.